examples/example_plugin/example_plugin/ui/index.js

```diff
@@ -1,11 +1,16 @@
 export { default as ExampleAppFullWidthPageComponent } from "./FullWidthPage"
+export { default as ExampleAppOverrideModelView } from "./ModelView"
 
 const app_config = {
     detail_tabs: {},
     full_width_components: {
-        "dcim:sites": ["ExampleAppFullWidthPageComponent"]
+        "dcim:devices": ["ExampleAppFullWidthPageComponent"]
     },
-    view_overrides: {}
+    view_overrides: {
+        "example-plugin:other-models": {
+            "retrieve": "ExampleAppOverrideModelView"
+        },
+    }
```

Comment:
Can you just add one paragraph to our existing docs to this convention? Not perfect but _starts_ getting these conventions documented.